Add notice to deployment guides coming soon (#985)
surrealdb/docs.surrealdb.com: src/content/doc-surrealdb/deployment/digitalocean.mdx

# Deploy on Digital Ocean

> [!NOTE]

> > This deployment guide isnot yet available. In the meantime, please look at other deployment guides to guide you through deploying SurrealDB in different environments.

Reviewer: ```suggestion
> This deployment guide is not yet available. In the meantime, please look at [our available deployment options](/docs/surrealdb/deployment) to guide you through deploying SurrealDB in different environments.
```
